src/Rules/RulePattern.php

```diff
@@ -70,12 +69,12 @@ public function getTokensOffsetRelativeToStartToken($startTokenName): array
             throw new ConfigurationException('No start token found for pattern '.print_r($this, true));
         }
 
-        $numTokens = count($this->tokens);
-
-        $offsets = array_map(static function ($idx) use ($startTokenIdx) {
-            return $idx - $startTokenIdx;
-        }, range(0, $numTokens - 1));
+        $offsets = array_map(
+            static fn (int $idx) => $idx - $startTokenIdx,
+            range(0, count($this->tokens) -1)
+        );
 
+        /** @phpstan-ignore-next-line */
         return array_combine($offsets, $this->tokens);
```

Comment:
@bcremer I'm getting this error on that line:

```
 ------ --------------------------------------------
  Line   src/Rules/RulePattern.php
 ------ --------------------------------------------
  78     No error to ignore is reported on line 78.
 ------ --------------------------------------------

 [ERROR] Found 1 error
```

Removing the `@phpstan-ignore-next-line` line fixes it. PHPStan version is 1.2.0. Can you confirm this?

Reply:
Cannot confirm, maybe a cache issue with phpstan on my side.

![image](https://user-images.githubusercontent.com/55820/146523850-35e4550a-112b-42da-913b-aa5161b2cdc1.png)

Reply:
Ah got it.

The error depends on the PHP-Version you are running phpstan with.

When running phpstan with PHP 8.0 no errors will be reported, when running with PHP 7.4 it will emit the error as seen above.

Reply:
```
parameters:
    phpVersion: 70400 # PHP 7.4
```